filters/shedder/admission_test.go

```diff
@@ -594,6 +598,127 @@ func TestAdmissionControlCleanupMultipleFilters(t *testing.T) {
 	}
 }
 
+type testTracer struct {
```

Comment:
This looks borrowed from https://github.com/zalando/skipper/pull/1964

Lets think ho we can extract it into e.g. tracing/tracingtest package

Reply:
Yes, it was copied from there.

Reply:
Addressed in this commit, see https://github.com/zalando/skipper/pull/3318/commits/bdea461506235c79299994c135a632cd35902862 .